docs/snippets/InferenceEngine_QueryAPI7.cpp

```diff
@@ -0,0 +1,12 @@
+#include <ie_core.hpp>
+
+int main() {
+using namespace InferenceEngine;
+//! [part7]
+InferenceEngine::Core core;
+auto network = core.ReadNetwork("sample.xml");
+auto exeNetwork = core.LoadNetwork(network, "GPU");
+std::map<std::string, uint64_t> statistics_map = core.GetMetric("GPU", GPU_METRIC_KEY(MEMORY_STATISTICS));
```

Comment:
BTW I am curious that is this really added to InferenceEngine_Query ? 
Seems that currently MEMORY_STATISTICS is only available as **GPU_**METRIC_KEY.

Reply:
@vladimir-paramuzov  How do you think the location of this metric?
I am putting the GPU specific metric to GPU.md, as follows:
https://github.com/openvinotoolkit/openvino/pull/8226/files

Reply:
@yeonbok I like the approach with GPU specific metrics in GPU.md

Reply:
@yeonbok @vladimir-paramuzov Let me move this update for document and code snippet to GPU.md

Reply:
done